design/src/production-chains/index.md

```diff
@@ -0,0 +1,137 @@
+Instead, it's more useful to:
+
+1. Consider smaller parts of the graph at a time.
+2. Reason abstractly about the properties of these graphs, and their effect on gameplay.
+
+Let's go through some of the relevant properties of production graphs!
+
+### Number of nodes (vertex count)
+
+This is a simple measure of the complexity of the game's crafting system.
+If there are more items to find and make, there's more game to both learn and play.
+
+Increasing this value increases playtime and diversity at the cost of complexity and development costs.
+
+### Number of [connected components](https://en.wikipedia.org/wiki/Component_(graph_theory))
+
+Imagine the recipes as physically connecting various parts of your graph, and then try to pull it apart.
+How many parts can you split it into without breaking any of the edges?
+
+The number of connected components is *very* high relative to the number of nodes in most farming games, survival crafters and other simple crafting systems.
+This means that you don't need to reason too hard about what else you could use your resources for,
+but limits the [cohesion](../glossary.md#cohesion) of the game's various systems and resource trees.
+
+### Distance between nodes
+
+This describes how many steps it takes to get from a desired set of raw resources to an end product.
+When considered globally, this is the **graph diameter.**
+
+This is often very low in simple crafting games, capping out at one or two.
+Colony sims generally have chains of length one to three,
+and even most factory builders end up with a chain length of one to four.
+
+Increasing this has a massive complexity cost, but adds a lot of [depth](../glossary.md#depth-and-complexity).
+Increasing this well (from a game-design perspective) requires some combination of:
+
+- simpler, less effective early recipes to slowly teach the player
+  - this is the strategy taken by the AngelBob's mod pack for Factorio: iron plates can be produced in many different ways, and only slowly becomes more complex
+- good uses for intermediate steps in the chain, either as end products, or via heavy branching
+  - this is also used in AngelBob's: sulfur is complex to make but exceedingly useful
+- exceptionally good tooling that displays the entire production tree needed for each recipe
+- exceptional patient and/or masochistic players
+  - this is the strategy taken by [Pyanodon's mod pack for Factorio](https://www.reddit.com/r/factorio/comments/yd1i7l/finally_after_20_hours_of_work_i_present_the/)
+
+### Number of incoming edges
+
+How many different ways can I make this item?
+In simple crafting games, this is almost always one, or a class of equivalent goods (different types of metal are common) can be substituted.
+
+Even in factory builders, this can often be quite small: one is common for base gameplay, and even modded rarely has more than three or four options.
+
+When this is 1 everywhere, the recipe graph is a tree.
```

Comment:
If we wanna be precise about this, it can also be a forest (or rather, a polyforest, because it's a directed graph), because the graph not necessarily connected: <https://en.wikipedia.org/wiki/Tree_(graph_theory)>